csrc/scheduler/transpose.cpp

```diff
@@ -127,6 +127,27 @@ void TransposeScheduler::computeHeuristics(
 
 namespace {
 
+// propagation could miss trivial reduction iterdomain on input tensors, since
+// that doesn't have any dependency and doesn't map to anything, we can naively
+// just reorder them so they won't interfere with tiling.
+// See https://github.com/NVIDIA/Fuser/issues/1659#issuecomment-1907053830
+void cleanInnerNDLeafDomain(TensorView* tv, int n) {
```

Comment:
What does `ND` mean here?

Reply:
I meant to say N-Dimension... I guess I should have just left it as `cleanInnerLeafDomain`

Reply:
Should this be called `moveReductionsOut`?